README.md

```diff
@@ -15,6 +19,8 @@ The [`legacy` branch](https://github.com/okta/okta-sdk-dotnet/tree/legacy) conta
 
 Need help? Contact [developers@okta.com](mailto:developers@okta.com) or use the [Okta Developer Forum].
 
+See this project's documentation on the [API Reference section](https://developer.okta.com/okta-sdk-dotnet/latest/api/index.html).
```

Comment:
I think we should make a new section `## API reference` above `## Installation` and say:

You can browse the [SDK reference documentation] to understand what classes and methods are available, Or, read below to see how to accomplish common tasks.

What do you think @laura-rodriguez?